addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3662,12 +3662,9 @@ var SnippetsMenu = Widget.extend({
         // snippet will leave the tooltip "forever" visible even if the "Hide in
         // Desktop" button is gone.
         const tooltipClass = 'aria-describedby';
-        const tooltippedEl = this.el.querySelector(`[${tooltipClass}^="tooltip"]`);
-        if (tooltippedEl) {
-            const tooltipEl = document.getElementById(tooltippedEl.getAttribute(tooltipClass));
-            if (tooltipEl) {
-                Tooltip.getInstance(tooltipEl).hide();
-            }
+        const tooltipEl = this.el.querySelector(`[${tooltipClass}^="tooltip"]`);
+        if (tooltipEl) {
+            Tooltip.getInstance(tooltipEl).hide();
```

Comment:
Indeed the `tooltipEl` is not found here. It's because in Email Marketing it seems that the document on which we call `getElementById()` is the wrong one and so we don't find the tooltip. The fix here should be about being sure we call `getElementById()` on the correct document. I think that the tooltip is always displayed in the same document of the tooltipped element. So here a good solution would be to just change the way we find the tooltipEl and do something like that:
```js
const tooltipEl = tooltippedEl.ownerDocument.getElementById(tooltippedEl.getAttribute(tooltipClass));
```

cc @dmo-odoo

Reply:
If I understood correctly, you confirmed that the current code in production has an issue in Email Marketing. I understand your fix suggestion and I think it is likely to work. However, I was expecting you to tell me why you thought the proposed changes in this PR wouldn't work in your case, since you seemed to need `getElementById` for some reason but I don't understand why, and the PR proposes to remove it.

Reply:
Sorry, I was too quick on this one, in fact, the current code works correctly and in addition it removes code so it's better :+1: . I thought that `getInstance` should be called with the `tooltipEl` as parameter and not with the `tooltippedEl`. So I'd just change the variable name here to `tooltippedEl`

Reply:
I'm confused. The PR already proposes to rename `tooltippedEl` to `tooltipEl`. Did I miss something ?

Reply:
![image](https://github.com/odoo/odoo/assets/78849981/ddd66e3b-2ef6-45bd-b98a-4a47093cdd61)
IMO `A` should be called `tooltipEl` and `B` should be called `tooltippedEl`

Reply:
@shsa-odoo Could you adapt the variable name please ?

Reply:
@dmo-odoo so i need to change tooltipEl  to  tooltippedEl, right!

Reply:
@shsa-odoo Yes, thanks !

Reply:
@dmo-odoo done!